comet/utils/debrid.py

```
@@ -25,101 +29,150 @@ async def cache_availability(debrid_service: str, availability: list):
-                timestamp = EXCLUDED.timestamp
-            """
-            await database.execute_many(query, no_season_episode_values)
-    else:
-        query = """
-            INSERT 
-            INTO debrid_availability
-            VALUES (:debrid_service, :info_hash, :file_index, :title, :season, :episode, :size, :parsed, :timestamp)
-        """
-        await database.execute_many(query, values)
+            await database.execute_many(query, values)
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)